databus/src/main/java/com/bazaarvoice/emodb/databus/db/cql/CqlSubscriptionDAO.java

```diff
@@ -36,6 +36,10 @@
 
     private static final String CF_NAME = "subscription";
 
+    // Currently, by default, subscription ttl limit is set to 365 days (in seconds),
+    // but that could be changed in future
+    private final static int SUBSCRIPTION_TTL_LIMIT = Math.toIntExact(Duration.ofDays(365).getSeconds());
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Reorder the modifiers to comply with the Java Language Specification. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.qa.bazaarvoice.com/coding_rules#rule_key=squid%3AModifiersOrderCheck)

Reply:
fixed